Graphics/GraphicsEngineOpenGL/src/GLContextLinux.cpp

```diff
@@ -166,12 +166,14 @@ void GLContext::SwapBuffers(int SwapInterval)
         {
             glXSwapIntervalEXT(display, wnd, SwapInterval);
         }
-#endif
-#if GLX_EXT_swap_control && GLX_MESA_swap_control
-        else
-#endif
-#if GLX_MESA_swap_control
-            if (glXSwapIntervalMESA != nullptr)
+#    if GLX_MESA_swap_control
+        else if (glXSwapIntervalMESA != nullptr)
+        {
+            glXSwapIntervalMESA(SwapInterval);
+        }
+#    endif
+#elif GLX_MESA_swap_control
+        if (glXSwapIntervalMESA != nullptr)
```

Comment:
Will this not make both `glXSwapIntervalEXT` and `glXSwapIntervalMESA ` being called if they both loaded at run time? I believe this branch is not needed at all since, like you said, both extensions are always statically enabled in GLEW

Reply:
No?
It's #elif(else if),